### PR TITLE
First working version

### PR DESCRIPTION
**Pull request by brisvag**
Better classification, images are working, some gui attempts.

### CODE REVIEW

**Comment by brisvag** [comment]
Once we add test, maybe this should be out version 0.1? We can then push to master and party :P

File: peepingtom/core/base.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
+from abc import ABC, abstractmethod
```
Comment (alisterburt) [inline peepingtom/core/base.py:1]: I think with the current structure Model should go here too, what do you think?

**Review by alisterburt — APPROVED** [review]
looks great! merge merge merge

**Comment by alisterburt** [comment]
yeah sure! let's try to get it done over the weekend?

**Pull request merged**